File: TTT Python PySimpleGui/model/Spielstein.py
from model.Farbe import Farbe

class Spielstein(object):
    def __init__(self, Farbe: Farbe = None, ID: int = -1):
        self._Farbe = Farbe
        self._id = ID

    @property
    def id(self) -> int:
        return self._id

    @property
    def Farbe(self) -> Farbe:
        return self._Farbe

File: TTT VS WindowsFramework/model/Spielstein.cs
namespace model
{
    public class Spielstein : ISpielstein
    {
        Farbe Farbe;
        int ID;
        public Spielstein(Farbe Farbe, int ID)
        {
            this.Farbe = Farbe;
            this.ID = ID;
        }
        public Farbe getSteinfarbe()
        {
            return this.Farbe;
        }
        public int getSteinID()
        {
            return this.ID;
        }
    }
}
